sites/fast-website/webpack.common.js

```diff
@@ -37,42 +36,40 @@ module.exports = {
             {
                 test: /\.css$/,
                 exclude: /node_modules/,
-                use: [
-                    {
-                        loader: MiniCssExtractPlugin.loader,
-                        options: {
-                            esModule: false,
-                        },
-                    },
-                    {
-                        loader: "css-loader",
-                    },
-                ],
+                loader: "css-loader",
             },
             {
                 test: /\.svg$/,
                 type: "asset/source",
             },
-            {
-                test: /\.ejs$/,
-                use: [
-                    {
-                        loader: "ejs-loader",
-                        options: {
-                            esModule: false,
-                        },
-                    },
-                ],
-            },
```

Comment:
Remove deprecated loader.
The html-bundler-webpack-plugin support the `ejs` "out of the box".